src/SSVOpenHexagon/Core/MenuGame.cpp

```diff
@@ -1686,6 +1688,16 @@ void MenuGame::update(ssvu::FT mFT)
 
     hg::Joystick::update();
 
+    if(hg::Joystick::nextPackRisingEdge())
+    {
+        changePackAction(1);
+    }
+    else if(hg::Joystick::previousPackRisingEdge())
+    {
+        changePackAction(-1);
+    }
+    focusHeld = hg::Joystick::focusPressed();
```

Comment:
```suggestion
    }
    
    focusHeld = hg::Joystick::focusPressed();
```

I like vertical space :)